doc/manual/contracts.md

```diff
@@ -378,8 +377,7 @@ error: Non mergeable terms
 **Warning: `=` vs `|`**
 
 It may be tempting to use `=` instead of  `|` to attach a record contract to a
-field. That is, writing `Contract = {foo = {bar | Str}}` instead of `Contract =
-{foo | {bar | Str}}`. When applying this contract, the merging operator will
+field. That is, writing `Contract = {foo = {bar | Str}}` instead of `Contract = {foo | {bar | Str}}`. When applying this contract, the merging operator will
 apply the `Str` contract to the field `foo` of the checked value. At first
 sight, `=` also fits the bill. However, there are a number of subtle but
 potentially surprising differences.
```

Comment:
Trying to maintain the wrapping.

```suggestion
field. That is, writing `Contract = {foo = {bar | Str}}` instead of
`Contract = {foo | {bar | Str}}`. When applying this contract, the merging
operator will apply the `Str` contract to the field `foo` of the checked value.
At first sight, `=` also fits the bill. However, there are a number of subtle
but potentially surprising differences.
```